src/components/client/notifications/MarkdownComponents.tsx

```
@@ -0,0 +1,22 @@
+import React from "react";
+
+export const MarkdownComponents = {
+  h1: ({ node, ...props }: any) => (
```

ESLint

[error] 'node' is defined but never used.

(@typescript-eslint/no-unused-vars)